compiler/merge_test.cc

```diff
@@ -70,5 +76,91 @@ TEST(MergeTest, PadConv) {
     graph.CheckSanity("merged");
 }
 
+TEST(MergeTest, TransposeGemmA) {
+    Type type(Dtype::kFloat32, {});
+    Graph graph("test");
+    Value* input = graph.AddInputValue("input", type);
+    Value* output = graph.AddOutputValue("output", type);
+
+    std::string trans_name;
+    {
+        GraphBuilder gb(&graph, "test", input);
+
+        // Transpose node
+        Value& trans = *gb.Op(Node::kTranspose, {input});
```

Comment:
[optional] Please use pointers if you change https://github.com/pfnet-research/chainer-compiler/pull/334/files#diff-f3b721c5d8105e1df0219d67e6c87f99R183